package androidx.compose.material.icons.outlined;

import bj.n;
import g1.m0;
import g1.t;
import j0.b;
import k0.a;
import k0.c;
import k1.e;
import k1.f;
import k1.h0;
import kotlin.jvm.internal.l;

/* JADX INFO: compiled from: r8-map-id-cf66f36031e302b9a7b5a76da57d208ae1b6ba4a38bba3bee1d4ed6983c54df3 */
/* JADX INFO: loaded from: classes.dex */
public final class NoteKt {
    private static f _note;

    public static final f getNote(b bVar) {
        l.f("<this>", bVar);
        f fVar = _note;
        if (fVar != null) {
            return fVar;
        }
        e eVar = new e("Outlined.Note", 24.0f, 24.0f, 24.0f, 24.0f, 0L, 0, false, 224);
        int i10 = h0.f9067a;
        m0 m0Var = new m0(t.f6905b);
        n nVarS = a.s(16.0f, 4.0f, 4.0f);
        nVarS.i(-1.1f, 0.0f, -2.0f, 0.9f, -2.0f, 2.0f);
        nVarS.t(12.01f);
        nVarS.i(0.0f, 1.1f, 0.9f, 1.99f, 2.0f, 1.99f);
        nVarS.k(16.0f);
        nVarS.i(1.1f, 0.0f, 2.0f, -0.9f, 2.0f, -2.0f);
        c.x(nVarS, -8.0f, -6.0f, -6.0f);
        k0.f.z(nVarS, 4.0f, 18.01f, 6.0f, 11.0f);
        k0.b.C(nVarS, 5.0f, 5.0f, 7.01f, 4.0f);
        e.a(eVar, nVarS.f3166a, 0, m0Var);
        f fVarB = eVar.b();
        _note = fVarB;
        return fVarB;
    }
}
